pkg/command/root.go

```diff
@@ -210,6 +215,18 @@ func newRootCmd() *cobra.Command {
 	return rootCmd
 }
 
+// SetLoggerVerbosity sets the verbosity of the logger if TANZU_CLI_LOG_LEVEL is set
+func SetLoggerVerbosity() {
```

Comment:
No need to make this public, right? `SetLoggerVerbosity` -> `setLoggerVerbosity`

Reply:
changed to private method